src/board/v3/nap/nap_common.c

```diff
@@ -119,3 +124,55 @@ static void nap_exti_thread(void *arg)
   }
 }
 
+u8 nap_conf_rd_version_string(char version_string[])
+{
+  u8 i = 0;
+  u32 reg = 0;
+
+  do {
+    NAP->CONTROL = (i << 1);
```

Comment:
Also, is there anything else in the config register besides the offset for DNA / version?